### DISCUSSION TITLE
Developing rnote.

### DISCUSSION THREAD

**Discussion by Partha-Sarker** [flxzt/rnote#176] [Development]
I am having trouble running the application while in development. Do I have to build every time I make a change? Or Is there any efficient way to see the changes in action? And it takes a really long time to build the application with 

`flatpak-builder --user flatpak-app build-aux/com.github.flxzt.rnote.Devel.json`  command.

**Comment by flxzt** [comment]
You'd have to build once you made a change. Personally I use the flatpak extension in vscode, which caches and does incremental builds, which only takes a couple of seconds compared to minutes for a full build.

**Reply by Partha-Sarker** [reply]
But when I try to build using vscode flatpak extension, the terminal shows

`flatpak build-init /my_project_directory/rnote/.flatpak/repo com.github.flxzt.rnote.Devel org.gnome.Sdk org.gnome.Platform 42`

and nothing happens after.

**Reply by flxzt** [reply]
you need to run `update dependencies` and then `build and run the application` again after initializing.

**Reply by Partha-Sarker** [reply]
Can you tell me the command to update dependencies?

**Reply by flxzt** [reply]
check out vscodes command palette:
![Bildschirmfoto vom 2022-05-17 13-34-01](https://user-images.githubusercontent.com/19841886/168801836-89c4ee58-7b59-4d6f-b394-e5c742500c14.png)

**Reply by Partha-Sarker** [reply]
Thanks for the help. I am able to build the application really fast now.
